Data/Sys/GameSettings/SD2J01.ini

```diff
@@ -26,8 +26,8 @@ PH_ZNear =
 PH_ZFar =
 
 [Video_Settings]
-SafeTextureCacheColorSamples = 512
```

Comment:
Just benchmarked here and the performance impact seems much lower than what I thought. Will set Safe Texture Cache for Just Dance 3 as well since the intro logo also drop a frame at the beginning...